src/jira-connector.ts

```diff
@@ -22,26 +23,36 @@ export class JiraConnector {
     });
   }
 
-  async getTicketDetails(key: string): Promise<JIRADetails> {
-    const issue: JIRA.Issue = await this.getIssue(key);
-    const {
-      fields: { issuetype: type, project, summary },
-    } = issue;
-
-    return {
-      key,
-      summary,
-      url: `${this.JIRA_BASE_URL}/browse/${key}`,
-      type: {
-        name: type.name,
-        icon: type.iconUrl,
-      },
-      project: {
-        name: project.name,
-        url: `${this.JIRA_BASE_URL}/browse/${project.key}`,
-        key: project.key,
-      },
-    };
+  async getTicketDetails(key: string): Promise<void> {
+    try {
+      const githubConnector = new GithubConnector();
+      const issue: JIRA.Issue = await this.getIssue(key);
+      const {
+        fields: { issuetype: type, project, summary },
+      } = issue;
+
+      const details = {
+        key,
+        summary,
+        url: `${this.JIRA_BASE_URL}/browse/${key}`,
+        type: {
+          name: type.name,
+          icon: type.iconUrl,
+        },
+        project: {
+          name: project.name,
+          url: `${this.JIRA_BASE_URL}/browse/${project.key}`,
+          key: project.key,
+        },
+      };
+      await githubConnector.updatePrDetails(details);
+    } catch (error) {
+      if (error.response) {
+        console.log('Error code - ' + error.response.status);
```

Comment:
would not this error be logged once more in global catch block?

Reply:
Yea, I just saw the output, its redundant. Removed this log.